src/Character/Character.jsx

```diff
@@ -89,12 +89,10 @@ export default function Character() {
   };
 
   const renderClassOptions = () => {
-    const classList = [];
     const classNames = Object.keys(classes);
-    for (let cName of classNames) {
-      classList.push(<option key={sha256(cName)}>{capitalize(cName)}</option>);
-    }
-
+    const classList = classNames.map((className) => (
```

Comment:
I realize now that I'm looking at this a second time that `className` is _sort of_ a keyword. It means something very specific in React (the actual class that will be applied to a DOM Element).

Should maybe consider renaming this to make it clear that this is specific to the Elden Ring class.

Reply:
Ooh good catch

Reply:
class too cuz of Class(), maybe classLabel? classType?

Reply:
`classType` seems fine to me!